static/resources/json/estimated_log_usage_dashboard_configuration.json

```diff
@@ -241,6 +241,7 @@
                         "q": "top(sum:datadog.estimated_usage.logs.ingested_events{datadog_is_excluded:false,$service,$index} by {datadog_index}.as_count(), 50, 'sum', 'desc')"
                     }
                 ],
+                "custom_links":[{"link":"https://dd.datad0g.com/logs?index={{datadog_index.value}}","label":"See Logs in Explorer"}],
```

Comment:
Hi @NoisomePossum Should these be `app.datadoghq.com` links instead of `dd.datadog.com`?